src/components/Organisms/CardList/CardList.component.js

```diff
@@ -16,7 +16,9 @@ const CardList = ({ category, children, url, src, categoryDescription }) => (
   <Section className={`${styles.list} ${styles[category]}`}>
     <header className={styles.header}>
       <a href={url} className={styles.logoLink}>
-        <img src={src} alt={categoryDescription} className={styles.logo} />
+        {src && (
```

Comment:
This smells like another oopsie! Good catch. If it's not required, make sure you know what happens when it isn't there :smile: